### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

**Comment by Holzhaus** [comment]
Merge?

File: source/manual_guidelines.rst [hunk 1]
```diff
@@ -1,272 +0,0 @@
-.. include:: /shortcuts.rstext
-
-Guidelines for Mixxx Manual writers
```
Comment (daschuer) [inline source/manual_guidelines.rst:3]: Where else is this info?

Comment (Holzhaus) [reply source/manual_guidelines.rst:3]: We could put it in the wiki or a Readme file

Comment (daschuer) [reply source/manual_guidelines.rst:3]: Maybe a readme file is not accessible ..

Comment (Holzhaus) [reply source/manual_guidelines.rst:3]: It's certainly more accessible than a file that is hidden in the source directory and isn't even part of any toctree. I think GitHub supports `rst` files, I'll just move it into the repo root and link it from the readme.

Comment (Holzhaus) [reply source/manual_guidelines.rst:3]: Done.

**Review by daschuer — APPROVED** [review]
LGTM, thank you.

**Pull request merged**